wiki/Beatmap/en.md

```diff
@@ -1,38 +1,52 @@
+*Main article: [Difficulties](/wiki/Difficulties)*
 
-From the auditory perspective, a beatmap can augment the song by providing one or multiple sets of audio samples, or [hitsounds](/wiki/Beatmapping/Hitsound), that serve as an additional layer of feedback.
+A difficulty is a file which describes the placement of hit objects, hitsounds, and special effects like [kiai time](/wiki/Kiai_time). It also contains [difficulty settings](/wiki/Beatmap_Editor/Song_Setup#difficulty) and other parameters that directly affect gameplay. Difficulties of a beatmap have different structure and skill requirements and sometimes may be played only in one [game mode](/wiki/Game_mode).
 
-Visual components of a beatmap, contrary to its patterns, provide auxiliary experience during gameplay and can be disabled in the [visual settings](/wiki/Visual_Settings) overlay for the most part. Those include:
+## Beatmap submission
 
-- [background imagery](Background) and video, which both can act as a playfield;
-- [kiai time](/wiki/Kiai_time) and [combo colours](/wiki/Glossary/Combo_colour) that help distinguish different beatmap sections from each other;
-- [storyboard](/wiki/Storyboards) that adds motion graphics and special effects, or provides a background story or theme for the song;
-- [custom skin](/wiki/Skinning), which changes the appearance of interface and gameplay elements;
-- [title text](Title_Text), which presents the song's metadata.
+*Main article: [Submission](/wiki/Submission)*
 
-## Beatmapsets
+Beatmap authors can [submit](/wiki/Glossary#bss) their creations to the public [beatmap listing](https://osu.ppy.sh/beatmapsets). While every beatmap is attributed to [a single person](/wiki/Beatmap/Beatmapsets/Beatmapset_host), it is often a joint effort: some difficulties may be mapped by other people working [in collaboration](/wiki/Beatmap/Beatmap_collaborations) or [separately](/wiki/Beatmap/Beatmapsets/Guest_difficulty).
 
-*Main article: [Beatmapsets](Beatmapsets)*
+<!-- TODO: after https://github.com/ppy/osu-web/issues/5852 is resolved, this section will need an update -->
 
-On a higher level, beatmaps are grouped into [beatmapsets](Beatmapsets), and then [submitted](/wiki/Glossary#bss) to the [beatmap listing](https://osu.ppy.sh/beatmapsets). Each beatmapset typically covers one song and holds together several beatmaps with different object placement and gameplay settings. In this context, a beatmap is called a [difficulty](/wiki/Difficulties).
+After the submission, a beatmap gains additional metadata fields, such as description, language, genre, and the explicit content marker, which can be changed by the beatmap author themselves on the web site. The beatmap also acquires [title text](Title_Text), appearance of which may be altered with the aid of [NAT](/wiki/People/The_Team/Nomination_Assessment_Team).
```

Comment:
```suggestion
After the submission, a beatmap gains additional metadata fields, such as description, language, genre, and the explicit content marker, which can be changed by the beatmap author themself on the website. The beatmap also acquires [title text](Title_Text), appearance of which may be altered with the aid of [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team).
```

Reply:
not really, that's still [themselves](https://english.stackexchange.com/questions/216617/the-hidden-flaw-in-singular-they-what-to-do-about-reflexive-pronouns). for NAT, I'll go ahead and refer to the whole team instead of a single person, so 50-50